docs/source/release/v6.5.0/diffraction.rst

```diff
@@ -10,30 +10,26 @@ Powder Diffraction
 
 New features
 ############
-- The ISIS powder diffraction scripts support the PaalmanPings correction method,
-  calling :ref:`PaalmanPingsMonteCarloAbsorption <algm-PaalmanPingsMonteCarloAbsorption>`.
-  To access it, set ``empty_can_subtraction_method: 'PaalmanPings'`` in the user config file.
-  Also there is new optional config settings (with their default values)
-  ``paalman_pings_events_per_point: 1000``
-- It is now possible to exclude specific banks from the total scattering merge banks for Polaris using -1 in the q_lims. For more details see :ref:`create_total_scattering_pdf<create_total_scattering_pdf_polaris-isis-powder-ref>`.
-- Added Total radial distribution function (G_k(r)) option to :ref:`PDFFourierTransform v2 <algm-PDFFourierTransform-v2>` .
-- Add a switch to allow an absolute normalization to be performed by the ISIS powder diffraction scripts when running on the POLARIS instrument. As part of this the setting "crystal_density" has been removed from the SampleDetails class. The density of the material can now be set instead by supplying two of "number_density", "number_density_effective" and "packing_fraction"
+- The ISIS powder diffraction scripts support the PaalmanPings correction method, calling :ref:`PaalmanPingsMonteCarloAbsorption <algm-PaalmanPingsMonteCarloAbsorption>`. To access it, set ``empty_can_subtraction_method: 'PaalmanPings'`` in the user config file. Also, there is new optional config settings (with their default values) ``paalman_pings_events_per_point: 1000``.
+- It is now possible to exclude specific banks from the total scattering merge banks for Polaris using -1 in the ``q_lims``. For more details see :ref:`Polaris.create_total_scattering_pdf<create_total_scattering_pdf_polaris-isis-powder-ref>`.
+- Added Total radial distribution function (:math:`G_k(r)`) option to :ref:`PDFFourierTransform v2 <algm-PDFFourierTransform-v2>`.
+- Added a switch to allow an absolute normalization to be performed by the ISIS powder diffraction scripts when running on the POLARIS instrument. As part of this the setting ``crystal_density`` has been removed from the SampleDetails class. The density of the material can now be set (instead by supplying two of ``number_density``, ``number_density_effective`` and ``packing_fraction``).
```

Comment:
```suggestion
- Added a switch to allow an absolute normalization to be performed by the ISIS powder diffraction scripts when running on the POLARIS instrument. As part of this the setting ``crystal_density`` has been removed from the SampleDetails class. The density of the material can now be set by supplying two of ``number_density``, ``number_density_effective`` and ``packing_fraction``.
```
A stupidly minor nitpick from me here, but the bit in brackets is probably the more important part of this sentence! I don't think the word instead helps much.

Reply:
I was unsure if the scentance was saying; _Now the density can be set; whereas in the past you had to suply two of..._, or if it was saying what you've correct it to. Does your correction match how that process functions now?

Reply:
Yea you could always set the density but this sentence is explaining how it should now be done. Yes my correction matches what happens now. :-)